src/components/LegalDisclaimer.vue

```diff
@@ -35,3 +35,13 @@ export default {
   },
 };
 </script>
+<style>
+  li:before{
```

Comment:
I think you should probably use https://css-tricks.com/almanac/properties/l/list-style/ for this

Reply:
I followed the documentation of `list-style` property in Mozilla as well [1] but it seems like there's no list-style color tag hence, to change the color I had used `li:before`. There can be one way to add the color which is to use a span tag to wrap the text around the list item. Do you want me to take that approach? 

Thanks for the review!
[1]: https://developer.mozilla.org/en-US/docs/Web/CSS/list-style